### ISSUE TITLE
Enable CORS support

### ISSUE THREAD

**Issue by cmungall** [broadinstitute/broad-translator#19]
CORS support isn't enabled. @newgene is updating the smartAPI docs to indicate that this should be done.

It's not clear if this is a major issue so long as the intent is not to have javascript clients connect directly.

It causes a minor annoyance because if I go here:

http://smart-api.info/ui/?url=/api/metadata/89590198e93fb9377ab547eb903c5532#/default/get_modelList

And try Execute it says 

![image](https://user-images.githubusercontent.com/50745/31913179-4a37c56c-b7fb-11e7-8b8d-6bacb9426e0f.png)

This is easily circumvented by going directly to the URL, bypassing the swagger UI: https://translator.ncats.io/broad-pgm-translator/modelList

However, it is a minor annoyance. Not clear if there are other implications

**Comment by newgene** [comment]
Right. If you want your API to be called in a browser via AJAX call (e.g. like the swagger UI API documentation page does), your API needs to support CORS. Depending on the web framework, you should be able to find relevant instruction via google.

**Comment by cmungall** [comment]
Section on CORS added: https://github.com/NCATS-Tangerine/translator-api-registry#cors-support
